src/data-mgt/node/package.json

```diff
@@ -5,6 +5,7 @@
     "main": "./bin/www",
     "scripts": {
         "start": "node ./bin/www",
+        "stage": "NODE_ENV=staging node ./bin/www",
```

Comment:
new script for staging.
start script (default one) is for production.

The rest of the scripts are for development purposes.